packages/engine-formula/src/functions/text/find/__test__/index.spec.ts

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright 2023-present DreamNum Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { describe, expect, it } from 'vitest';
+
+import { FUNCTION_NAMES_TEXT } from '../../function-names';
+import { Find } from '../index';
+import { NumberValueObject, StringValueObject } from '../../../../engine/value-object/primitive-object';
+import { ArrayValueObject, transformToValue, transformToValueObject } from '../../../../engine/value-object/array-value-object';
+import { ErrorType } from '../../../../basics/error-type';
+
+describe('Test find function', () => {
+    const findFunction = new Find(FUNCTION_NAMES_TEXT.FIND);
+
+    describe('Find', () => {
+        it('Find text in single cell', () => {
+            const findText = StringValueObject.create('Univer');
+            const withinText = StringValueObject.create('Hello Univer');
+            const result = findFunction.calculate(findText, withinText);
+            expect(result.getValue()).toStrictEqual(7); // Indexing is 1-based
```

Comment:
Still get ArrayValueObject here, you should use `transformToValue(result.getArrayValue())` to get array result